ci/spack.yaml

```diff
@@ -0,0 +1,12 @@
+# Spack environment file to build library dependencies such as NetCDF and GSI-ncdiag
+spack:
+  packages:
+    all:
+      compiler: [intel, gcc]
+  specs:
+  - netcdf-c@4.7.4
+  - netcdf-fortran@4.5.3
+  - w3emc@2.9.2
+  view: true
+  concretizer:
+    unify: when_possible
```

Comment:
I would put this as `true` so that the whole environment is built as a single coherent set. Good in a CI setting where you'll have no duplicate packages.